Documentation/Helm-Charts/operator-chart.md

```diff
@@ -41,102 +41,103 @@ For example settings, see the next section or [values.yaml](https://github.com/r
 
 The following tables lists the configurable parameters of the rook-operator chart and their default values.
 
-| Parameter                           | Description                                                                                                                                      | Default                                                   |
```

Comment:
looks like some many changes in this one which is not expected.

Reply:
Markdown all in one extension formatted it on saving file & this is the preview
![Screenshot from 2022-09-20 17-19-48](https://user-images.githubusercontent.com/23611106/191250250-aa72fe84-4e77-4443-8347-25132b100010.png)

Reply:
> looks like some many changes in this one which is not expected.
most of these are the removal of spaces or addition of `---` which I think came due adding new field